components/utils/MeasureUploadHeader.tsx

```diff
@@ -0,0 +1,45 @@
+import { Text, Popover, List, Anchor, ActionIcon } from '@mantine/core';
+import React, { useState } from 'react';
+import { InfoCircle } from 'tabler-icons-react';
+
+export default function MeasureUploadHeader() {
+  const [opened, setOpened] = useState(false);
+
+  return (
+    <div
```

Comment:
Is this div with these styles really necessary? This seems like exactly what a Mantine `Group` component would do. Maybe we can replace it with that?

Reply:
Nice catch